front/app/modules/commercial/content_builder/admin/components/CraftComponents/Iframe/index.tsx

```diff
@@ -44,6 +44,9 @@ const Iframe = ({ url, height, hasError, title }: Props) => {
   );
 };
 
+const supportArticleUrl =
+  'https://support.citizenlab.co/en/articles/6354058-embedding-elements-in-the-content-builder-to-enrich-project-descriptions';
+
```

Comment:
Are support articles always in English? Or should the support article URL be to the same language as the platform? Just not quite sure how that works :)

Reply:
Good question - the link should also be added to the translations so that it can be internationalised! :) Also - if it changes for whatever reason, it can be updated via Crowdin and not directly in the code.

Reply:
Good point 👏